packages/driver/src/cross-origin/communicator.ts

```diff
@@ -55,6 +56,10 @@ export class PrimaryOriginCommunicator extends EventEmitter {
         data.data = reifySnapshotFromSerialization(data.data as any)
       }
 
+      if (data?.event.includes(SNAPSHOT_EVENT_PREFIX) && !Cypress._.isEmpty(data?.data)) {
```

Comment:
Any reason to not combine this if with the above?

```javascript
      if (FINAL_SNAPSHOT_EVENT === data?.event
       || (data?.event.includes(SNAPSHOT_EVENT_PREFIX) && !Cypress._.isEmpty(data?.data))
      ) {
```

Reply:
Put everything snapshot related into a single prefix in the communicator in 044f09bbef